entries/p/plutus.it.json

```
@@ -0,0 +1,14 @@
+{
+   "Site Name":{
+      "domain":"plutus.it",
+      "img":"plutus.it.png",
+      "tfa":[
+         "totp"
+      ],
+      "documentation":"https://support.plutus.it/hc/en-us/articles/7203346943517-Account-creation-password-reset-2FA-and-account-block-suspension",
+      "categories":[
+         "banking",
+         "finance"
```

I think these categories are a better fit for the service.
```suggestion
         "cryptocurrencies",
         "banking"
```

---

Agreed